src/bindings/python/src/pyopenvino/graph/dict_attribute_visitor.cpp

```diff
@@ -274,6 +274,18 @@ void util::DictAttributeSerializer::on_adapter(const std::string& name, ov::Valu
     if (m_attributes.contains(name)) {
         OPENVINO_THROW("No AttributeVisitor support for accessing attribute named: ", name);
     }
+    
+    if (auto _adapter = dynamic_cast<ov::AttributeAdapter<std::shared_ptr<ov::op::util::Variable>>*>(&adapter)) {
+        m_attributes[name.c_str()] = _adapter->get()->get_info().variable_id;
+    } else if (auto _adapter = dynamic_cast<ov::AttributeAdapter<ov::PartialShape>*>(&adapter)) {
+        auto partial_shape = _adapter->get();
+        py::list shape;
+        for (const auto& dim : partial_shape) {
+            dim.get_length();
+            shape.append(dim.is_dynamic() ? py::int_(-1) : py::int_(dim.get_length()));
```

Comment:
```suggestion
            shape.append(dim.is_dynamic() ? py::int_(-1) : py::int_(dim.get_length()));
```
Same here, does it have to be `py::int_`?

Reply:
As mentioned above, the `PartialShape` object might have "dynamic" dimension, i want to use number `-1` to represent dynamic dimensions, so i implemented it in that way.